src/main/scala/org/openeo/opensearch/backends/STACClient.scala

```diff
@@ -69,6 +70,7 @@ class STACClient(private val endpoint: URL=new URL("https://earth-search.aws.ele
   override def getCollections(correlationId: String = ""): Seq[Feature] = {
     val getCollections = http(s"$endpoint/collections")
       .option(HttpOptions.followRedirects(true))
+      .timeout(connTimeoutMs = 10000, readTimeoutMs = 40000)
```

Comment:
This is already covered by the change in OpenSearchClient.

Reply:
fixed